src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/IHttpHandler.cs

```diff
@@ -0,0 +1,41 @@
+// <copyright file="IHttpHandler.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Net.Http;
+using System.Threading;
+
+namespace OpenTelemetry.Exporter.OpenTelemetryProtocol.Implementation
+{
+    /// <summary>
+    /// Interface partially exposing <see cref="HttpClient"/> methods.
+    /// </summary>
+    public interface IHttpHandler
```

Comment:
Does this need to be public? Probably needs to be moved to a different namespace if so.

Reply:
It's part of implementation at the moment. Will make it internal.